zenodo/modules/records/serializers/schemas/common.py

```diff
@@ -244,11 +245,39 @@ def validate_data(self, data):
             )
 
 
-class AlternateIdentifierSchemaV1(IdentifierSchemaV1):
+class ResourceType(Schema):
```

Comment:
```suggestion
class ResourceTypeMixin(object):
```

Having this as a mixin will help with inheritance problems